cmd/destroy/cmddestroy.go

```diff
@@ -66,7 +66,7 @@ func (r *DestroyRunner) RunE(cmd *cobra.Command, args []string) error {
 
 	// Run the destroyer. It will return a channel where we can receive updates
 	// to keep track of progress and any issues.
-	ch := r.Destroyer.Run()
+	ch := r.Destroyer.Run(cmd, args)
```

Comment:
We should not pass cobra commands into the code that lives in `/pkg`

Reply:
I think I've fixed this. The ManifestReader.Read() is now in the cmd like the other commands. Have a look